README.md

```diff
@@ -1,152 +1,187 @@
 
-```console
-$ make bin/app; make execute
-```
+   ```
+   // Linux & macos
+   $ ./scripts/<PLATFORM>/install.sh
 
-#### Windows 
-```shell
-> mingw32-make bin/app && mingw32-make execute
-```
+   // Windows
+   > ./scripts<PLATFORM>/install.bat
+   ```
 
+5. Duplicate the `.env.template` file and rename it to `.env`.
+6. Uncomment and fill in the following variables. Each variable should point to a directory or file:
 
-Using this method can save you a huge amount of time compiling *(in reality, just a few seconds)* each time you make a small change to your code! If you want to know more about how it works, you should have a read through [the docs entry explaining the Makefile](https://github.com/CapsCollective/raylib-cpp-starter) from another of our template projects.
+   1. `VK_LAYER_PATH` - if you're using validation layers, set this to the directory where they're located.
+   2. `VK_ICD_FILENAMES` - if using MacOS, set this to the path that the MoltenVK ICD is located. This usually points to a file called `MoltenVK_icd.json`.
+   3. `VULKAN_INCLUDE_DIR` - point this to the directory where the vulkan headers are located.
 
-While separate compilation works quite well in most scenarios, it's not magic, and there are a few caveats to take note of here:
+7. Save the .env file. This should be enough to build the project.
 
-1. Changing `.h` files will often result in longer compile times by causing all files that include them to recompile
-2. Constant changes to files included by many others in your program (like a base-class) will also cause all of those dependent to recompile
-3. Including widely-scoped files (like the whole of `vulkan/vulkan.h`) will add all of its own includes as dependent and increase the build time
-4. Placing includes in `.h` files instead of forward-declarations will also increase recursive includes and therefore the build time
+#### Setup without the Vulkan SDK
 
-### Passing Args to the Executable
-For working with some projects, you may want to pass arguments to the program once it's been built. This can be achieved by assigning values to the `ARGS` flag in the `Makefile` like below:
+1. Navigate to the project root.
+2. Run the install script for your platform:
 
-#### macOS & Linux
+   ```bash
+   // Linux & macos
+   $ ./scripts/install.sh
 
-```console
-$ make ARGS="--somearg"
-```
-#### Windows
-```shell
-> mingw32-make ARGS="--somearg"
-```
+   // Windows
+   > ./scripts/install.bat
+   ```
 
-### Specifying Custom Macro Definitions
-You may also want to pass in your own macro definitions for certain configurations (such as setting log levels). You can pass in your definitions using the `MACRO_DEFS` flag:
+3. This should install all required dependencies. Once completed a `.env` file will be generated with all required variables. If the build is completed with no issue then you can proceed to build the project.
 
+#### Setup without the Vulkan SDK with validation layers
```

Comment:
I would write this up as an extension or bonus to the previous section

Reply:
Done.